libs/ui/src/ContestTally.tsx

```diff
@@ -86,14 +86,52 @@ const ContestTally = ({
           }
         })
 
-        const { contest, tallies, metadata } = finalContestTally
+        const { tallies, metadata } = finalContestTally
 
         const talliesRelevant = precinctId
           ? districts.includes(contest.districtId)
           : true
 
         const { ballots, overvotes, undervotes } = metadata
-        const options = getContestOptionsForContest(contest as AnyContest)
+
+        const contestOptionTableRows = []
```

Comment:
I know TS can infer it from the `.push` calls, but I prefer being explicit. Your call.
```suggestion
        const contestOptionTableRows: JSX.Element[] = []
```